onnx/external_data_helper.py

```diff
@@ -96,7 +98,7 @@ def set_external_data(tensor,  # type: TensorProto
 def convert_model_to_external_data(model, all_tensors_to_one_file=True, location=None):
     # type: (ModelProto, bool, Optional[Text]) -> None
     """
-    call to set all tensors as external data. save_model saves all the tensors data as external data after calling this function.
+    Call to set all tensors as external data. save_model saves all the tensors data as external data after calling this function.
```

Comment:
Would help to clarify in the documentation that only raw_data tensors will be transformed, not typed tensors.